patches/minecraft/net/minecraft/world/chunk/storage/ExtendedBlockStorage.java.patch

```diff
@@ -9,3 +18,46 @@
          IBlockState iblockstate = this.func_177485_a(p_177484_1_, p_177484_2_, p_177484_3_);
          Block block = iblockstate.func_177230_c();
          Block block1 = p_177484_4_.func_177230_c();
+@@ -63,7 +67,7 @@
+ 
+     public boolean func_76663_a()
+     {
+-        return this.field_76682_b == 0;
++        return this.field_76682_b == 0 && this.blocklightCount == 0 && (this.skylightCount == 0 || this.skylightCount == 15 * 16 * 16 * 16); //Forge: Take light into account. Fixes MC-116690
```

Comment:
i'd say put the comment on the variables instead of here so we can just look at the variable and see what all 5 of these patch sites are for